### ISSUE TITLE
getting actual translations in test time

### ISSUE THREAD

**Issue by yftah89** [lvapeab/nmt-keras#28]
I have a trained model and now I want to use it to translate.
In the config file I changed MODE from training to sampling and RELOAD to my best epoch model.
The testing seems to going fine but all I get as an output is val.coco file.
Is there some way to get the actual translations in addition? like in the training process in which I get the actual translations on the validation set? 

Thanks in advanced, this github is wonderful by the way :)

**Comment by lvapeab** [comment]
Hi @yftah89,

you should use the [sample_enseble.py](https://github.com/lvapeab/nmt-keras/blob/master/sample_ensemble.py) script, for translating new data. [Here](https://github.com/lvapeab/nmt-keras/blob/master/examples/documentation/ensembling_tutorial.md) you have some information about how to use it, basically, you must set the dataset, source data and model(s) to use. 

Cheers.
